frontend/src/components/employees/employees.module.scss

```diff
@@ -0,0 +1,7 @@
+.employees {
+    background-color: olive;
+    color: orange;
+    grid-area: main;
+    width: 100%;
+    height: 100%;
+}
```

Comment:
if we do not switch to HOCs or some way to reduce code repetition, maybe we should use a mixin to avoid repeating lines 4-6?

Reply:
As above, these are going to change once the components are properly implemented. 
If there is repetition in the real thing we can always refactor later.

Reply:
It feels very likely that the repetition will remain. Especially if we have a title/header. If people feel strongly I will remove my changes requested but it will be a lot easier to reaactor now. Do we have designs for what the components will contain?